### PR TITLE
add failover transport example

### PR DESCRIPTION
**Pull request by luu-alex**
created an example for the failover transport 👍

### CODE REVIEW

**Review by buffalojoec — COMMENTED** [review]
I think this looks good. @steveluscher anything you want added?

File: packages/rpc-transport-http/README.md [hunk 1]
```diff
@@ -216,5 +216,28 @@ async function retryingTransport<TResponse>(...args: Parameters<RpcTransport>):
...
+
+// A failover transport that will try each transport in order until one succeeds before failing
+async function failoverTransport<TResponse>(...args: Parameters<RpcTransport>): Promise<TResponse> {
+    let requestError;
+
+    for (const transport of transports) {
+        try {
+            return await transport(...args);
+        } catch (err) {
+            console.error(err);
```
Comment (steveluscher) [inline packages/rpc-transport-http/README.md:237]: Love it. I think you meant to set `requestError` here.
```suggestion
            requestError = err;
            console.error(err);
```

**Review by steveluscher — CHANGES_REQUESTED** [review]
bow-bowing

File: packages/rpc-transport-http/README.md (hunk 1 above)
Comment (luu-alex) [reply packages/rpc-transport-http/README.md:238]: you read my mind!

**Comment by luu-alex** [comment]
Thank you!

**Pull request merged**